PACKAGES

```diff
@@ -90,7 +91,8 @@ lwt-dllist
 magic-mime
 markup
 merlin
-mirage.3.6.0
+metrics
+mirage
```

Comment:
The version pin for mirage was there just to make sure we don't get a less fresh version due to some other constraint. Need to figure out a better solution...